blocks/editable/index.js

```diff
@@ -103,6 +107,23 @@ export default class Editable extends wp.element.Component {
 		this.props.onFocus();
 	}
 
+	isActive() {
+		return document.activeElement === this.editor.getBody();
+	}
+
+	onSelectionChange() {
+		// We must check this because selectionChange is a global event.
+		if ( ! this.isActive() ) {
+			return;
+		}
+
+		const content = this.getContent();
+
+		this.setState( {
```

Comment:
I'm not sure if this would trigger a re-render if the value is the same?

> This performs a shallow merge of stateChange into the new state.
https://facebook.github.io/react/docs/react-component.html#setstate

I do notice a lot of re-renders form node change though, not from this one.